appeals/api/src/server/endpoints/integrations/integrations.utils.js

```diff
@@ -0,0 +1,38 @@
+/** @typedef {import('pins-data-model').Schemas.AppellantSubmissionCommand['documents'][number]} AppellantSubmissionDocument */
+/** @typedef {import('pins-data-model').Schemas.LPAQuestionnaireCommand['documents'][number]} LPAQuestionnaireCommandDocument */
+
+/**
+ *
+ * @param {(AppellantSubmissionDocument|LPAQuestionnaireCommandDocument)[]} documents
+ * @returns {(AppellantSubmissionDocument|LPAQuestionnaireCommandDocument)[]}
+ */
+export const renameDuplicateDocuments = (documents) => {
+	const seen = new Set();
+
+	return documents.map((document) => {
+		let key = `${document.documentType}_${document.originalFilename}`;
+
+		if (!seen.has(key)) {
+			seen.add(key);
+			return document;
+		}
+
+		const nameParts = document.originalFilename.split('.');
+		const extension = nameParts.pop() || '';
+		const baseName = nameParts.join('.');
+		let counter = 1;
+		let newFilename;
+
+		do {
+			newFilename = `${baseName}_${counter}.${extension}`;
```

Comment:
do we add `copy_${counter}` for clarity?

Reply:
Hey Ben, sorry what do you mean here? to add 'copy' to the name? I thought would be less disruptive to just append the suffix, rather than changing the name even more...

Reply:
Sorry yes I was wondering if we added "copy" to the name to make it clearer for users?